PythonTool/TagExpansion.py

```diff
@@ -168,20 +178,42 @@ def expand_vector_tags(mesh_file, main_dir_name = None):
        LookupError: If the file does not contain any vector tags.
     """
 
+    # Create a dictionary with the MB element types and their integer values.
+    elements = {
```

Comment:
I'm hesitant about this. PyMOAB already has these types defined, right? Can we simply rely on those instead of dictionary values?

Reply:
Unless of course they aren't working correctly... :grin:

Reply:
I decided to do it this way because when you call `types.MBHEX` or any other element type, it is evaluated as an integer! And if I tried to append a string onto `types.MB____`, it is not read correctly. Therefore, I needed a dictionary with the integer corresponding to each element type in order to call `get_entities_by_type` correctly.

Reply:
Maybe it's better to use the pyMOAB defined values as the values in this dictionary:
e.g. `"vertex" : types.MBVERTEX, "edge" : types.MBEDGE` etc

Reply:
I agree and the code now reflects that!